src/tests/units/operator_tests.cpp

```diff
@@ -297,8 +297,10 @@ void tests(DeviceType device, Type floatType = Type::float32) {
     s2->val()->get(values); CHECK(values == vS2);
 
     CHECK(m3->val()->scalar() == 9);
-
-    s4->val()->get(values); CHECK(std::equal(values.begin(), values.end(), vS4.begin(), floatApprox));
+  
+    s4->val()->get(values); 
+    std::cout << values[0]<<" "<<values[1]<<std::endl;
```

Comment:
Please remove this artifact.

Reply:
Incidentally this is also where the merge conflict sits.

Reply:
741cd86